RocketCast/XMLParser.swift

```diff
@@ -10,92 +10,165 @@ import Foundation
 
 class XMLParser: NSObject {
     
-    var strXMLData:String = ""
-    var currentElement:String = ""
-    var passData:Bool=false
-    var passName:Bool=false
+    var parser = NSXMLParser()
+    var element = NSString()
     
+    var listOfEpisodes = [EpisodeModel]()
     
-    var url: NSURL?
-    var xmlContent:NSDictionary?
-    init(url: NSURL) {
-        super.init()
-        self.url = url
-        self.xmlContent = parseData()
-    }
+    var episodeTitle = NSMutableString()
```

Comment:
There's a more OOP way of doing this by generate objects as you go through the XML